src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/TargetsTest.java

```diff
@@ -510,4 +510,291 @@ public void testNoOCPTargetHotplugFlagSupportedOS() {
+        Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("None".toLowerCase())));
+        Assert.assertFalse(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("OpenJDK".toLowerCase())));
+    }
+
+    @Test
+    public void testGivenRHELOracleJDK8_ThenOpenJDKRecommendedTarget() {
+        Map<String, Object> facts = new HashMap<>();
+        // always add a String fact with the name of the agenda group defined in the DRL file (e.g. "SourceCosts")
+        facts.put("agendaGroup", "Targets");
+
+        WorkloadInventoryReportModel workloadInventoryReportModel = new WorkloadInventoryReportModel();
+        workloadInventoryReportModel.setOsFamily("RHEL");
+        workloadInventoryReportModel.addWorkload("Oracle JDK 8");
+
+        facts.put("workloadInventoryReportModel",workloadInventoryReportModel);
+
+        Map<String, Object> results = createAndExecuteCommandsAndGetResults(facts);
+
+        Assert.assertEquals(3, results.get(NUMBER_OF_FIRED_RULE_KEY));
+        Utils.verifyRulesFiredNames(this.agendaEventListener,
+                "AgendaFocusForTest",
+                "Target_None",
+                "Target_OpenJDK"
+        );
+
+        List<WorkloadInventoryReportModel> reports = Utils.extractModels(GET_OBJECTS_KEY, results, WorkloadInventoryReportModel.class);
+
+        // just one report has to be created
+        Assert.assertEquals(1, reports.size());
+        WorkloadInventoryReportModel report = reports.get(0);
+        Assert.assertNotNull(report.getRecommendedTargetsIMS());
+        Assert.assertEquals(2, report.getRecommendedTargetsIMS().size());
+        Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("None".toLowerCase())));
+        Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("OpenJDK".toLowerCase())));
+    }
+
+    @Test
+    public void testGivenRHELOracleJDK11_ThenOpenJDKRecommendedTarget() {
+        Map<String, Object> facts = new HashMap<>();
+        // always add a String fact with the name of the agenda group defined in the DRL file (e.g. "SourceCosts")
+        facts.put("agendaGroup", "Targets");
+
+        WorkloadInventoryReportModel workloadInventoryReportModel = new WorkloadInventoryReportModel();
+        workloadInventoryReportModel.setOsFamily("RHEL");
+        workloadInventoryReportModel.addWorkload("Oracle JDK 11");
+
+        facts.put("workloadInventoryReportModel",workloadInventoryReportModel);
+
+        Map<String, Object> results = createAndExecuteCommandsAndGetResults(facts);
+
+        Assert.assertEquals(3, results.get(NUMBER_OF_FIRED_RULE_KEY));
+        Utils.verifyRulesFiredNames(this.agendaEventListener,
+                "AgendaFocusForTest",
+                "Target_None",
+                "Target_OpenJDK"
```

Comment:
The same comments apply as in #64 re: "Target_None" and other targets co-existing. Also, as the changes for that PR are also here, do we need to merge #64 at all?

Reply:
@m-brophy this should be solved after merging https://github.com/project-xavier/xavier-analytics/pull/64. This PR is based on https://github.com/project-xavier/xavier-analytics/pull/64